src/screens/EventDetailsScreen/component.js

```diff
@@ -20,7 +21,6 @@ import text from "../../constants/text";
 import type { Event, LocalizedFieldRef } from "../../data/event";
 import locale from "../../data/locale";
 import chevronLeftWhite from "../../../assets/images/chevron-left-white.png";
-import heartWhite from "../../../assets/images/heart-white.png";
```

Comment:
Can we delete this icon or do we still use it somewhere else?

Reply:
It can go. Nice catch.